butterfree/hooks/schema_compatibility/cassandra_table_schema_compatibility_hook.py

```diff
@@ -0,0 +1,58 @@
+"""Cassandra table schema compatibility Hook definition."""
```

Comment:
Considering that we're working on migrations' automation, I think we won't need these hooks anymore in the future. For now, we can keep them, but I think we should remove these schema compatibility hooks in the near future. What do you think?

Reply:
I think so :thinking: :thought_balloon: